.github/workflows/pytest_slow.yml

```diff
@@ -7,6 +7,9 @@ on:
   push:
     branches: ["master", "release-*"]
     tags: ["v*.*.*"]
+  # just for testing, will be removed so that it only runs on commit to master.
```

Comment:
nit: probably don;'t need this comment

Reply:
Yes, put it there so that reviewers know what it's for

Reply:
oh I see, I get it now